.devcontainer/Dockerfile

```diff
@@ -81,3 +98,20 @@ RUN make install-kubebuilder
 RUN make install-kustomize
 RUN make install-test-dependency
 ENV SHELL /bin/bash
+
+# Add useful aliases
+RUN echo "alias k=kubectl" >> "/root/.bashrc"
+# Add autocomplete to kubectl
+RUN echo "source <(kubectl completion bash)" >> "/root/.bashrc"
+RUN echo "source <(kubectl completion bash | sed 's/kubectl/k/g')" >> "/root/.bashrc"
+# Add kubectx 
+RUN git clone https://github.com/ahmetb/kubectx.git /root/.kubectx \
```

Comment:
The experience of switching between Kube contexts is not as smooth as you would hope for.
 Kind replaces the current kubeconfig file, the only way you can switch back and forth between kind and other clusters (k8s, minishift, minikube,...)  kubeconfig is changing the value of environment variable from
KUBECONFIG="/root/.kube/kind-config-kind" to mounted path which user has.
We haven't mounted the kubeconfig on the user's local machine as far as I remember

Reply:
Yeah the reason I installed was to use the `kubens` commands to easy switch namespaces while I was working on things, the context stuff worked alright for me I just by flipping the `KUBECONFIG` env var as you suggest :)

Reply:
agreed on using `kubens` , can you please explain why do we need?

```
RUN echo "source <(kubectl completion bash)" >> "/root/.bashrc"
RUN echo "source <(kubectl completion bash | sed 's/kubectl/k/g')" >> "/root/.bashrc"
```
Couldn't we define alias for kubectl in  "/root/.bashrc"?